participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java

```diff
@@ -8,7 +8,15 @@
 
 package com.google.cloud.healthcare.fdamystudies.config;
 
+import org.springframework.context.annotation.Bean;
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Modified copyright text